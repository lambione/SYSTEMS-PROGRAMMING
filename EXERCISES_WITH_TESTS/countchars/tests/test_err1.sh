#!/bin/sh
exec "$1" xyz tests/f1 tests/fx1 tests/f2

