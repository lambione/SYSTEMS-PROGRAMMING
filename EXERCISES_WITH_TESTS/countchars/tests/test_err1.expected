tests/f1 x:13 y:17 z:0
Error reading file tests/fx1
tests/f2 x:1 y:17 z:0
